### PR TITLE
"Array Iterator" never appear in codes

### PR DESCRIPTION
**Pull request by mfuji09**
> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'

> What was wrong/why is this fix needed? (quick summary only)

"Array Iterator" never appear in codes, so I think it shouldn't be styled as a code like `Array Iterator`.

> Anything else that could help us review it

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/array/index.md [hunk 1]
```diff
@@ -327,7 +327,7 @@ The properties and elements returned from this match are as follows:
 - {{jsxref("Array.prototype.join()")}}
   - : Joins all elements of an array into a string.
 - {{jsxref("Array.prototype.keys()")}}
-  - : Returns a new `Array Iterator` that contains the keys for each index in the array.
+  - : Returns a new Array Iterator that contains the keys for each index in the array.
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/array/index.md:330]: Let's uniformize this one too.
```suggestion
  - : Returns a new _array iterator_ that contains the keys for each index in the array.
```

File: files/en-us/web/javascript/reference/global_objects/array/values/index.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ browser-compat: javascript.builtins.Array.values
 {{JSRef}}
 
 The **`values()`** method returns a new
-**`Array Iterator`** object that contains the values for each
+**Array Iterator** object that contains the values for each
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/array/values/index.md:19]:
```suggestion
_array iterator_ object that contains the values for each
```

File: files/en-us/web/javascript/reference/global_objects/bigint64array/index.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ The **`BigInt64Array`** typed array represents an array of 64-bit signed integer
 - {{jsxref("TypedArray.join", "BigInt64Array.prototype.join()")}}
   - : Joins all elements of an array into a string. See also {{jsxref("Array.prototype.join()")}}.
 - {{jsxref("TypedArray.keys", "BigInt64Array.prototype.keys()")}}
-  - : Returns a new `Array Iterator` that contains the keys for each index in the array. See also {{jsxref("Array.prototype.keys()")}}.
+  - : Returns a new Array Iterator that contains the keys for each index in the array. See also {{jsxref("Array.prototype.keys()")}}.
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/bigint64array/index.md:73]:
```suggestion
  - : Returns a new _array iterator_ that contains the keys for each index in the array. See also {{jsxref("Array.prototype.keys()")}}.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Let's uniformize a little bit more.

**Comment by mfuji09** [comment]
@teoli2003 I agree!

**Comment by mfuji09** [comment]
@teoli2003 
And I have found more and I have fixed them too.

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks!

**Pull request merged**